R/utils.R

```diff
@@ -155,3 +155,86 @@ validate_data_has_expected_cols <- function(data,
   data_has_expected_columns <- all(expected_columns %in% colnames(data))
   stopifnot(data_has_expected_columns)
 }
+
+#' Checks data for missings and duplicates
+#'
+#' Applies consistency checks to data concerning the combinations of columns
+#' that should be unique in combination. In concrete:
+#'
+#' 1. it is checked if there are duplicate rows.
+#' 1. it is checked if there are duplicate rows on `composite_unique_cols`.
```

Comment:
2. not 1.?

Reply:
esoteric roxygen stuff. The second 1 will be translated to 2. in the manual. This way it is easier to change order or add elements